compiler-core/src/type_/tests/warnings.rs

```diff
@@ -437,39 +437,39 @@ fn unused_imported_module_with_alias_warnings_test() {
 #[test]
 fn unused_imported_module_no_warning_on_used_function_test() {
     assert_no_warnings!(
-        ("gleam/foo", "pub fn bar() { 1 }"),
+        ("thepackage", "gleam/foo", "pub fn bar() { 1 }"),
```

Comment:
I had to update these tests to pass in the package name due to changes with the macro.

The package name has not changed from what it was before.